index.js

```diff
@@ -88,11 +135,19 @@ Use the higher order function getAverageGoals to do the following:
  
 */
 
-function getAverageGoals(/* code here */) {
-    /* code here */
- }
+function getAverageGoals(cbFinals){
```

Comment:
Good job on this function! I know this passes the test they wrote but based on their description it sounds like they ask for two averages (Home and Away) but then the test is only checking one. 

Overall I think you wrote the code correctly in order to pass the test. I do think they either wrote their test wrong or need to change the wording of this problem in order to match with their test. 

YOU did a good job here with what you were given to check it.